receiver_init_commands:
#-----------------------------------------------------------------------------------------------------
# Default logging
- "LOG VERSION"

# Reset logging
# - "UNLOGALL COM1"

#-------------------------------------------------------------------------------------------------------
# Standard Logging Requests

# 在这里指定需要获取的数据，下面都已经注释掉了，可以按需要取消注释或者增加新的指令
#----------------------------
# Status & Config
# - "LOG INSCONFIGB ONCE"  # Log once to obtain initial configuration. Do not remove, needed for IMU messages.
# - "LOG TERRASTARINFOB ONCE"
# - "LOG TERRASTARSTATUSB ONCE"
# - "LOG TERRASTARINFOB ONTIME 300" # Periodic update.
# - "LOG TERRASTARSTATUSB ONTIME 300"
# - "LOG INSCONFIGB ONTIME 300" 
# - "LOG RXSTATUSB ONCHANGED"

# GNSS and INS Solutions

# Do not remove these messages -------------------------------------------------------------------------- 
# - "LOG BESTPOSB ONTIME 0.1" 
# - "LOG BESTVELB ONTIME 0.1" 
# - "LOG BESTUTMB ONTIME 1"
# - "LOG BESTGNSSPOSB ONTIME 0.1"
# - "LOG PPPPOSB ONTIME 1"
# - "LOG HEADING2B ONNEW"
# - "LOG PSRDOP2B ONCHANGED"
# - "LOG INSPVASB ONTIME 0.02"
# - "LOG INSPVAXB ONTIME 1"
# - "LOG CORRIMUDATASB ONNEW"
# - "LOG INSSTDEVB ONTIME 1"
# - "LOG RAWIMUSXB ONNEW"
#----------------------------------------------------------------------------------------------------------

# - "LOG TIMEB ONTIME 1" # Can be used as a 'heartbeat' message.

# Post-Processing Messages; not directly used to generate ROS messages.
# - "LOG INSUPDATESTATUSB ONNEW"
# - "LOG RAWEPHEMB ONNEW"
# - "LOG GLOEPHEMERISB ONNEW"
# - "LOG BDSEPHEMERISB ONNEW"
# - "LOG GALFNAVEPHEMERISB ONNEW"
# - "LOG GALINAVEPHEMERISB ONNEW"
# - "LOG RANGEB ONTIME 1"







